stripe.go

```diff
@@ -233,8 +278,8 @@ func (s *BackendConfiguration) NewRequest(method, path, key, contentType string,
 // the backend's HTTP client to execute the request and unmarshals the response
 // into v. It also handles unmarshaling errors returned by the API.
 func (s *BackendConfiguration) Do(req *http.Request, v interface{}) error {
-	if LogLevel > 1 {
-		Logger.Printf("Requesting %v %v%v\n", req.Method, req.URL.Host, req.URL.Path)
+	if s.LogLevel > 1 {
```

Comment:
We should also make helpers for `logInfo`, `logError`, `logDebug` so we don't have these ugly checks on LogLevel everywhere, but I'll tackle that in another PR.